src/wallet/keychain/wallet.rs

```diff
@@ -66,11 +67,11 @@ impl<T: ToInternalDescriptor + ToExternalDescriptor + Clone + Send + Sync + 'sta
     }
 
     #[instrument(name = "keychain_wallet.new_external_address", skip_all)]
-    pub async fn new_external_address(&self) -> Result<bdk::wallet::AddressInfo, BriaError> {
+    pub async fn new_external_address(&self) -> Result<AddressCreationInfo, BriaError> {
```

Comment:
I moved this bdk type from this function's return to be on the `AddressCreationInfo` entity as a From trait implementation. Is this correct?

Reply:
I ended up reverting this change because something else in the psbt_builder tests depended on the output from that changed function being a bdk type ([here](https://github.com/GaloyMoney/bria/blob/50193fbcb261b27e5ed3e5814a6b4e3d7e1c7010/tests/psbt_builder.rs#L58))